.gitignore

```
@@ -104,3 +104,4 @@ venv.bak/
 .mypy_cache/
 
 .DS_Store
+.vscode/settings.json
```

```suggestion
.vscode/
```

Should be safe to ignore the entire directory in this case.